src/C-DEngine/C-DViewModels/cde_HSI.cs

```diff
@@ -2331,7 +2373,8 @@
                 cdeMID = this.cdeMID,
                 ScopeIDHash = this.RealScopeID?.Substring(0, 4).ToUpper(),
                 References = this.References,
-                QStatus = this.QStatus
+                QStatus = this.QStatus,
+                IsOneWay = this.IsOneWay,
```

Comment:
Does this need the TSMFilter as well?

Reply:
I thought that the TSMFilter is semi-secret and it would be better not to show in public diagnostics (gives away information about potential attack surface).